tests/conftest.py

```diff
@@ -1447,3 +1449,23 @@ def password_or_none(request) -> Optional[str]:
     Note that this is a parametrized fixture.
     """
     return request.param
+
+
+def method_scope(cls):
+    """
+    Class decorator to make the setup/teardown to have the 'method' scope.
+
+    :param cls: the class. It must be a subclass of
+    :return:
+    """
+    enforce(
+        issubclass(cls, BaseAEATestCase),
+        "cannot use decorator if class is not instance of BaseAEATestCase",
+    )
+    old_setup_class = cls.setup_class
+    old_teardown_class = cls.teardown_class
+    cls.setup_class = classmethod(lambda _cls: None)
+    cls.teardown_class = classmethod(lambda _cls: None)
```

Comment:
what if two tests use the same class in one module, one with and one without decorator. Does it not lead to inconsistency?

Reply:
the decorator should be used only on concrete/final classes. I will put more info in the docstring